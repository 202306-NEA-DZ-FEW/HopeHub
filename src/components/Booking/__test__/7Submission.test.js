import renderer from "react-test-renderer";

import Submission from "../7Submission";

it("renders correctly", () => {
    const user = {
        totalTickets: 5, // Or any number you want for testing purposes
    };

    const tree = renderer.create(<Submission user={user} />).toJSON();
    expect(tree).toMatchSnapshot();
});
