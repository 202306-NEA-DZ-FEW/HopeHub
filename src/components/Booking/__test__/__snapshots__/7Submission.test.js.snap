// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-NeutralWhite dark:bg-Dark_Accent min-w-screen mb-12"
>
  <div
    className="w-full h-full px-8 lg:px-20 bg-NeutralWhite dark:bg-Dark_Accent flex flex-col"
  >
    <div
      className="mb-3 pt-12 font-poppins font-bold tracking-wideer text-NeutralBlack dark:text-NeutralWhite capitalize text-2xl lg:text-4xl leading-normal"
    >
      Submit your appointment
    </div>
    <div
      className="font-poppins font-regular text-justify text-NeutralBlack dark:text-NeutralWhite text-base lg:text-lg"
    >
      Click Submit if you are sure of all your choices.
    </div>
    <div
      className=" bg-NeutralWhite dark:bg-Dark_Accent lg:w-3/5 lg:h-1/2 sm:w-full sm:h-[80%] sm:leading-tight mx-auto mt-14 shadow-[4.0px_8.0px_8.0px_rgba(0,0,0,0.42)] rounded-lg relative'"
    >
      <div
        className="flex flex-col justify-center items-center text-center"
      >
        <h3
          className=" rounded-md py-6 text-NeutralBlack dark:text-NeutralWhite text-2xl font-semibold font-poppins capitalize"
        >
          Please be aware that this action
          <br />
          will cost you a ticket!
        </h3>
        <h3
          className="font-poppins text-NeutralBlack dark:text-NeutralWhite"
        >
          You currently have 
          5
           tickets.
        </h3>
      </div>
      <div
        className="flex justify-between "
      >
        <div
          className=" pl-6 py-10 lg:py-10 lg:pl-11 group "
        >
          <button
            className="w-28 h-10 rounded-md text-base font-poppins font-regular bg-Accent text-NeutralBlack dark:text-NeutralWhite dark:bg-Dark_Primary dark:hover:bg-[#3E4E68]  hover:bg-[#879AB8] hover:text-NeutralWhite hover:scale-105 duration-500"
          >
            Previous
          </button>
        </div>
        <div
          className=" pr-6 py-10 lg:py-10 lg:pr-11 group"
        >
          <button
            className="w-28 h-10 rounded-md text-base font-poppins font-regular bg-Accent text-NeutralBlack dark:text-NeutralWhite dark:bg-Dark_Primary dark:hover:bg-[#3E4E68]  hover:bg-[#879AB8] hover:text-NeutralWhite hover:scale-105 duration-500"
            onClick={[Function]}
          >
            Submit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
